dist/images/ovnkube.sh

```diff
@@ -126,6 +126,9 @@ ovn_nb_port=${OVN_NB_PORT:-6641}
 # OVN_SB_PORT - ovn south db port (default 6642)
 ovn_sb_port=${OVN_SB_PORT:-6642}
 
+ovn_hybrid_overlay_enable=${OVN_HYBRID_OVERLAY_ENABLE:${OVN_EXTENSIONS_ENABLE:-}}
```

Comment:
does it make sense to modify `ovnkube.sh` to support hybrid extensions given that we don't use `ovnkube.sh` and no one else is going to use hybrid extensions?

Reply:
I'm happy to drop this part if needed.

Reply:
But that said, this is an upstream PR, so having the script support is kinda necessary